lib/time/Kconfig

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2024 Bjarki Arge Andreasen
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig TIME
```

Comment:
Same here, the configuration should either be done in the RTC realm or refer to generic time concepts independent of a specific hardware type. This API is not general enough to represent general POSIX time.

Reply:
I will make an attempt to use this API as the basis for posix time, then we can discuss further. I will probably meet some of the hurdles you foresee :)